client/platform/desktop/backend/serializers/nist.ts

```diff
@@ -63,14 +63,14 @@ interface NistActivity {
 }
 
 function confirmNistFormat(data: NistFile) {
-  return data.activities && data.filesProcessed;
+  return !!(data.activities && data.filesProcessed);
```

Comment:
This is not equivalent.

`!!([]) === true` even though `[] == false`.

It's best not to rely on squishy truthiness and do strict null, undefined, or length assertions.